app/src/main/res/layout/activity_launch_screen.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We still have lots of hardcoded dimen values here.

Reply:
Done @oofoamalu